samples/tab-deeplink/nodejs/README.md

```diff
@@ -1,6 +1,6 @@
 ---
 page_type: sample
-description: Microsoft Teams sample app for demonstrating deeplink from Bot chat to Tab consuming Subentity ID
+description: Microsoft teams sample which displays deeplinks for call, video, chat, navigate to app and navigate within tab pages which are supported for tab and bot.
```

Comment:
Microsoft `Teams` 
update at all the places

Reply:
changed